README_developers.md

```diff
@@ -18,6 +18,48 @@ $ git checkout -b my-bug-fix
 - Keep pull requests focused on single issue
 - Whenever possible squash all commits on merge
 
+# Quickstart on CI Platforms
+
+If you are developing HiOp on one of our supported platforms, you may want to
+use the dependencies we have already installed there. In that case, you may
+source the respective variables script we have configured for that platform:
+
+```console
+$ cd /path/to/hiop
+$ export MY_CLUSTER=newell # or marianas or ascent
+$ source ./scripts/${MY_CLUSTER}Variables.sh
+$ mkdir build && cd build
+$ cmake ..
+$ ccmake . # Optional, if you would like to further configure the build
+$ make -j 16
+$ make test
+```
```

Comment:
Perhaps leaving `ccmake` outside the core instructions would better emphasize it is optional?
```suggestion
$ make -j 16
$ make test

Optionally, you may want to modify default CMake configuration with
`ccmake` or `cmake-gui` tools.
```